ibis/backends/base/sql/__init__.py

```diff
@@ -186,6 +198,9 @@ def execute(
 
         return result
 
+    def _register_in_memory_table(self, table):
+        pass
```

Comment:
Can you add a brief docstring here for future readers?

Also, this should probably raise `NotImplementedError`, since calling this with a no-op will lead to bugs downstream

```suggestion
        raise NotImplementedError
```